content/index/index.en.yaml

```diff
@@ -2,43 +2,41 @@ logoTitle: Ethereum Classic
 logoSubtitle: Build **Unstoppable** Applications
 overlayTitle: Ethereum Classic Summit
 overlaySubtitle: |
-  October 3rd & 4th, 2019  
+  October 3rd & 4th, 2019
```

Comment:
Did you mean to do this? In markdown a double space `  `  indicates a line break. Perhaps your editor accidentally did this?

(Please don't commit changes to whitespace and newlines etc.)

Reply:
Yeah its defintiely my editor. it's doen it to every document i've modified

Reply:
disabled in my editor. thanks for the help.